broker/Gemfile

```diff
@@ -65,7 +65,7 @@ end
 # Load extra gems specified in the broker configuration file
 conf_file_path = (ENV['OPENSHIFT_CONF_DIR'] || "/etc/openshift/") + 'broker.conf'
 if File.exists?(conf_file_path)
-  conf_file = File.open(conf_file_path)
+  conf_file = File.open(conf_file_path, 'rb')
```

Comment:
I tested this out and it seems to work OK on the weird EC2 node where we saw the problem.  I would _highly_ recommend putting a comment here.  This is still a bit of a mystery.  On my local dev system:

~~~
# file /etc/openshift/broker.conf
/etc/openshift/broker.conf: ASCII English text
~~~

On the system where it's "broken":

~~~
# file /etc/openshift/broker.conf
/etc/openshift/broker.conf: UTF-8 Unicode English text
~~~